cvpy/biomedimage/BiomedImage.py

```diff
@@ -328,3 +328,89 @@ def morphological_gradient(self, images: ImageTable, kernel_width: int = 3,
         conn.table.dropTable(morph_grad_2d)
 
         return ImageTable(morph_grad_3d)
+    
```

Comment:
Is Shruti going to work on adding the two methods to BiomedImageTable.py file? If yes, I will suggest you revert your changes on BiomedImage.py and BiomedImageTable.py files.

Reply:
I will take care of those 2 functions, Mu.